Update for latest EIP changes (#9)
lukasrosario/eip-5792-site: docs/pages/introduction.mdx

## What is EIP-5792?

> [EIP-5792](https://eips.ethereum.org/EIPS/eip-5792) adds new JSON-RPC methods to the wallet interface. These methods will enable applications to send multiple calls to a user's wallet. These calls can be enhanced by `capabilities` (such as paymasters), if they are supported by the wallet.

Reviewer: ```suggestion
[EIP-5792](https://eips.ethereum.org/EIPS/eip-5792) adds new JSON-RPC methods that enable applications to send multiple calls to a user's wallet. These calls can be enhanced by `capabilities` (such as paymasters), if they are supported by the wallet.
```

For both sides, capabilities establish a foundation for iteratively adding new functionality over time to further improve and streamline the user experience.

## Why now?

The approaching [Pectra](https://ethereum.org/en/roadmap/pectra) upgrade includes [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702), which will enable Externally Owned Accounts (EOAs) to set their address to be represented by a code of an existing smart contract. This means that any account on Ethereum will be able to make batched calls (assuming the EOA's designated smart contract supports it).

This is a transformative unlock for user experience, but for end-users to benefit, applications need to access that new capability. That is where EIP-5792 comes in.

The time is now!

## What is the status of this EIP?

This EIP is currently in "Review", and the team expect to move to "Final Call" soon. There is already significant support for the EIP across [wallets](/ecosystem/wallets) and [tools](/ecosystem/tools).

## How can I get involved?

> Review the [EIP](https://eips.ethereum.org/EIPS/eip-5792), join the conversation on [Ethereum Magicians](https://ethereum-magicians.org/t/eip-5792-wallet-function-call-api/11374) and join the [Telegram group](<TODO>).

Reviewer: lets remove the telegram todo for now

Author: Done
